src/translations/maps/linkedPhysicalIndexToValueMap.js

```diff
@@ -75,6 +76,22 @@ class LinkedPhysicalIndexToValueMap extends IndexMap {
     super.clear();
   }
 
+  /**
+   * Remove value for particular index.
+   *
+   * @param {number} physicalIndex Physical index.
+   */
+  clearValue(physicalIndex) {
```

Comment:
I haven't found any unit test for that method. Could I ask you to cover that?

Reply:
The `setValueAtIndex` already removes a value from the mentioned array.

https://github.com/handsontable/handsontable/blob/8ee91484fe1efa000c577faadbcdc11fac0d8f87/src/translations/maps/linkedPhysicalIndexToValueMap.js#L54-L60

However, the method does extra execution of `initValueOrFn` function or pass initial value to the `setValueAtIndex`.

I've used `clearValue` name as it works very similar to the current `clear` method.

Reply:
Added test within https://github.com/handsontable/handsontable/pull/7319/commits/c4091ccc7d255d07298631285aa5b3c8aa563b7d.